cluster/examples/kubernetes/ceph/object.yaml

```
@@ -51,7 +51,7 @@ spec:
     # The port that RGW pods will listen on (http)
     port: 80
     # The port that RGW pods will listen on (https). An ssl certificate is required.
-    securePort:
+    securePort: 0
```

In the example let's comment it out, but specify a real example port such as:
```suggestion
    # securePort: 443
```

---

Done.